qcsubmit/datasets.py

```diff
@@ -227,10 +233,11 @@ def add_molecule(self, molecule: off.Molecule):
                         )
             else:
                 # molecule already in list and coords not present so just return
-                return
+                return True
```

Comment:
Do you still need this or was this just for testing? If we need it please change the type hint to reflect this.

Reply:
Was for testing, but I don't see the harm in leaving since it lets me know if the molecule was deduplicated